flink-tests/src/test/java/org/apache/flink/test/classloading/ClassLoaderITCase.java

```diff
@@ -30,20 +30,20 @@
 import org.junit.Test;
 
 public class ClassLoaderITCase {
-	
+
 	private static final String INPUT_SPLITS_PROG_JAR_FILE = "target/customsplit-test-jar.jar";
 
 	private static final String STREAMING_PROG_JAR_FILE = "target/streamingclassloader-test-jar.jar";
 
 	private static final String KMEANS_JAR_PATH = "target/kmeans-test-jar.jar";
-	
+
 	@Test
 	public void testJobWithCustomInputFormat() {
 		try {
 			Configuration config = new Configuration();
 			config.setInteger(ConfigConstants.LOCAL_INSTANCE_MANAGER_NUMBER_TASK_MANAGER, 2);
 			config.setInteger(ConfigConstants.TASK_MANAGER_NUM_TASK_SLOTS, 2);
-			
```

Comment:
I know. I couldn't find the option to disable this in Intellij (clears white-space only lines).